migrations/versions/be9889a7cead_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: be9889a7cead
+Revises: e3caa0f2a16c
+Create Date: 2019-02-01 17:36:54.810899
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'be9889a7cead'
+down_revision = 'e3caa0f2a16c'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_featured', sa.Boolean(), nullable=True))
+    op.add_column('events_version', sa.Column('is_featured', sa.Boolean(), autoincrement=False, nullable=True))
```

Comment:
This should not be here

Reply:
@iamareebjamal So, I should try reverting the changes in schema, right?